spec/views/posts/show_spec.rb

```diff
@@ -24,7 +23,7 @@
   end

```

Comment:
- [ ] @team kindly change the title of this test to match what it's testing, displaying the author name